typescript/cli/src/deploy/warp.ts

```diff
@@ -130,12 +128,12 @@ async function runBuildConfigStep({
           ? base.address!
           : ethers.constants.AddressZero,
       owner,
-      mailbox: base.mailbox || mergedContractAddrs[baseChainName].mailbox,
```

Comment:
dont we have an address merging util?

Reply:
This is slightly different. it first checks the token config for overrides there, otherwise falls back to the merged address map